src/addons/transitions/ReactCSSTransitionGroupChild.js

```diff
@@ -111,24 +111,22 @@ var ReactCSSTransitionGroupChild = React.createClass({
     if (!this.timeout) {
       this.timeout = setTimeout(this.flushClassNameAndNodeQueue, TICK);
     }
-  },
+  };
 
-  flushClassNameAndNodeQueue: function() {
-    if (this.isMounted()) {
-      this.classNameAndNodeQueue.forEach(function(obj) {
-        CSSCore.addClass(obj.node, obj.className);
-      });
-    }
+  flushClassNameAndNodeQueue = () => {
+    this.classNameAndNodeQueue.forEach(function(obj) {
+      CSSCore.addClass(obj.node, obj.className);
+    });
```

Comment:
Yeah, this PR's diffs are created automatically by `react-codemod/transforms/class`.
I think we can do more refactoring.